### PR TITLE
[switch_config] Multiple configs per run

### PR DESCRIPTION
**Pull request by pablintino**

### CODE REVIEW

File: roles/switch_config/tasks/main.yml [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
-    switch_vars: "{{ (lookup('vars', inventory_hostname, default='')) }}"
+    switch_vars: >-
+      {{
+        lookup('vars', inventory_hostname, default={}) |
+        default(
+            (
+              (
+                (
+                  hostvars[inventory_hostname]["cifmw_switch_config_scenarios"]
+                  [cifmw_switch_config_scenario_name]| default({})
```
Comment (rebtoor) [inline roles/switch_config/tasks/main.yml:12]: care to improve spacing here?
```suggestion
                  [cifmw_switch_config_scenario_name] | default({})
```

Comment (pablintino) [reply roles/switch_config/tasks/main.yml]: done!

**Review by rebtoor — APPROVED** [review]

**Pull request merged**